android/res/layout/item_editor_input.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  style="@style/MwmWidget.Editor.MetadataBlock">
+
+  <ImageView
+    android:id="@+id/icon"
+    style="@style/MwmWidget.Editor.MetadataIcon"
+    tools:src="@drawable/ic_phone"/>
+
+  <android.support.design.widget.TextInputLayout
+    android:id="@+id/custom_input"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_alignParentBottom="true"
+    android:layout_centerVertical="true"
```

Comment:
Да, один лишний. Поправлю, спс.
> On Apr 21, 2016, at 4:19 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/item_editor_input.xml <https://github.com/mapsme/omim/pull/2947#discussion_r60577402>:
> 
> > +<RelativeLayout
> > +  xmlns:android="http://schemas.android.com/apk/res/android"
> > +  xmlns:tools="http://schemas.android.com/tools"
> > +  style="@style/MwmWidget.Editor.MetadataBlock">
> > +
> > +  <ImageView
> > +    android:id="@+id/icon"
> > +    style="@style/MwmWidget.Editor.MetadataIcon"
> > +    tools:src="@drawable/ic_phone"/>
> > +
> > +  <android.support.design.widget.TextInputLayout
> > +    android:id="@+id/custom_input"
> > +    android:layout_width="match_parent"
> > +    android:layout_height="wrap_content"
> > +    android:layout_alignParentBottom="true"
> > +    android:layout_centerVertical="true"
> А как эти два атрибута друг с другом дружат? Они же взаимоисключающие. Нет?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/d1348fd7592d78bb03e92481d414b5b5e2fe8bb6..21d88faa93db984e82364d89461ba8a3c9d1e6e7#r60577402>